hack/update-generated-completions.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+STI_ROOT=$(dirname "${BASH_SOURCE}")/..
```

Comment:
we don't have this `readonly` in any other script and I'm not going to change that here and not everywhere else.